app/delivery/send_to_providers.py

```diff
@@ -350,13 +350,13 @@ def provider_to_use(
 ) -> Any:
     """
     Get the provider to use for sending the notification.
-    SMS that are being sent with a dedicated number or to a US number should not use Pinpoint.
+    SMS that are being sent with a dedicated number or internationally should not use Pinpoint.
 
     Args:
         notification_type (str): SMS or EMAIL.
         notification_id (UUID): id of notification. Just used for logging.
         to (str, optional): recipient. Defaults to None.
-        international (bool, optional): Recipient is international. Defaults to False.
+        international (bool, optional): It is not clear what this flag indicates. Defaults to False.
```

Comment:
Wut?

Reply:
yeah... there are international numbers that aren't marked and toronto numbers that do have it set...

Reply:
```suggestion
        international (bool, optional):  Flags whether or not the message is outside of Canada and the US. Defaults to False.
```